user_guide/b2b_channels_index.rst

```diff
@@ -1,5 +1,5 @@
-More about B2B Channels
-=======================
+Business-to-business Sales
+==========================
 
 In the section, you can find more information about B2B channels and entities added to them by default.
```

Comment:
edit: In this section, you can find more information about B2B channels and the entities added to them by default.